packages/ra-ui-materialui/src/button/SaveButton.js

```diff
@@ -81,7 +82,7 @@ export class SaveButton extends Component {
                 className={classnames(classes.button, className)}
                 variant={variant}
                 type={type}
-                onClick={this.handleClick}
+                onClick={submitOnEnter ? undefined : this.handleClick}
```

Comment:
Does that mean that if submitOnEnter is true (which is the default), the save button doesn't work anymore? That sounds wrong.

Reply:
Nope, it fallback to the default html submit on enter, hence the necessity to have the `onSubmit` prop on the form

Reply:
Gothcha